utils/create_dummy_models.py

```diff
@@ -458,6 +479,18 @@ def convert_processors(processors, tiny_config, output_folder, result):
         result["warnings"].append(f"Failed to convert feature extractors: {e}")
         feature_extractors = []
 
+    if hasattr(tiny_config, "max_position_embeddings") and tiny_config.max_position_embeddings > 0:
```

Comment:
copied from `tests/pipelines/test_pipelines_common.py` on `main` (with some modification) - necessary to avoid some test failures.

The original one checks the following list - but I only have to check 2 model types. Some tests for other model types might be skipped (so I didn't detect any failure), and eventually it might be better to check if I have to include all the following model types.

```python
ROBERTA_EMBEDDING_ADJUSMENT_CONFIGS = [
    "CamembertConfig",
    "IBertConfig",
    "LongformerConfig",
    "MarkupLMConfig",
    "RobertaConfig",
    "XLMRobertaConfig",
]
```